src/posthog-core.ts

```diff
@@ -1271,6 +1289,24 @@ export class PostHog {
         }
     }
 
+    /**
+     * Sets properties for the Person associated with the current distinct_id.
+     *
+     *
+     * @param {Object} [userPropertiesToSet] Optional: An associative array of properties to store about the user
+     * @param {Object} [userPropertiesToSetOnce] Optional: An associative array of properties to store about the user. If property is previously set, this does not override that value.
+     */
+    setPersonProperties(userPropertiesToSet?: Properties, userPropertiesToSetOnce?: Properties): void {
+        if (!userPropertiesToSet && !userPropertiesToSetOnce) {
+            return
+        }
+
+        // Update current user properties
+        this.setPersonPropertiesForFlags(userPropertiesToSet || {})
```

Comment:
shouldn't this also update based on set once (looks like in the past we didn't account for them either, but I suspect we should cc @neilkakkar)?

Reply:
He said it shouldn't so I just trusted that 😅

Reply:
Nope, it shouldn't, because we can't tell on the client side if what we're setting once will actually be set once (i.e. if the prop already exists on our servers, then it will be wrong to override it here)